### ISSUE TITLE
Encode config struct into properties file

### ISSUE THREAD

**Issue by jacobfederer** [magiconair/properties#62]
Is there a way to change values in a config struct/use the default values and write them back into a properties file?

**Comment by aheling11** [comment]
+1

**Comment by einhverfr** [comment]
Yes please do this.  I might try to provide a PR to do this but I am less sure of my golang abilities.
